.cirrus.yml

```diff
@@ -1,5 +1,5 @@
 container:
-  image: smartrent/terraform-ci:1.1.3
+  image: smartrent/terraform-ci:1.5.7-1a
```

Comment:
If we're switching over to circle do we need to do this?  Can we just remove this file?

Reply:
yeah well need this until the PR merges and then we can move everything over to circle completely in a seperate PR